File: test/Basic/Str/ToLowerCase_20.php
<?php

namespace xltxlm\helper\test\Basic\Str;

use xltxlm\helper\Basic\Str;

/**
 *
 */
class ToLowerCase_20
{

    public function __invoke()
    {
        $news = (new Str("aBBc"))->ToLowerCase();
        if ($news != 'abbc') {
            throw new \Exception("error");
        }
    }

}

File: testgo/Basic/Str/ToLowerCase_20.go
package Str

import (
	"log"
	"libs/helper/go/Basic"
)

func TestToLowerCase_20() {
	log.SetFlags(log.Lshortfile | log.LstdFlags)
	news := Basic.NewStr("aBBc").ToLowerCase()
	if news != "abbc" {
		log.Fatal(news)
	}
	log.Print("ok")

}
